sdk/storage/azblob/blob/models.go

```diff
@@ -258,6 +258,32 @@ func (o *UndeleteOptions) format() *generated.BlobClientUndeleteOptions {
 
 // ---------------------------------------------------------------------------------------------------------------------
 
+type BlobDeleteType = generated.DeleteType
```

Comment:
Missing doc comment.  You'll also need to export the values for this `const` like [this](https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/storage/azblob/constants.go#L16-L19).